plutus-core/testlib/PlutusCore/Generators/PIR/Utils.hs

```diff
@@ -0,0 +1,75 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+module PlutusCore.Generators.PIR.Utils where
+
+import Prettyprinter
+
+import Data.Set (Set)
+import Data.Set qualified as Set
+import Data.Set.Lens (setOf)
+import Data.String
+import Test.QuickCheck
+
+import PlutusCore.Default
+import PlutusCore.MkPlc hiding (error)
+import PlutusCore.Name
+import PlutusCore.Pretty
+import PlutusIR
+import PlutusIR.Compiler.Datatype
+import PlutusIR.Subst
+
+import PlutusIR.Core.Instance.Pretty.Readable
+
+-- | Show a `Doc` when a property fails.
+ceDoc :: Testable t => Doc ann -> t -> Property
```

Comment:
So, hedgehog has these sorts of functions, maybe we should copy their naming? https://hackage.haskell.org/package/hedgehog-0.6.1/docs/Hedgehog-Internal-Property.html#v:annotate

Reply:
> So, hedgehog has these sorts of functions, maybe we should copy their naming? https://hackage.haskell.org/package/hedgehog-0.6.1/docs/Hedgehog-Internal-Property.html#v:annotate

The types are wildly different (like, take a different number of arguments and do a different thing), so I'm gonna bother trying to establish a connection.